### PR TITLE
101 doc length

### PR DESCRIPTION
**Pull request by thanasions**

### CODE REVIEW

**Comment by IanGrimstead** [comment]
This is odd - grumbles about commented-out-code are marked as resolved but not showing up in the diff as fixed? If you want them for a reason then fair enough but want to make sure they're not left by accident :)

**Comment by thanasions** [comment]
looks like it. Try now, just pushed them

File: scripts/algorithms/tfidf.py [hunk 1]
```diff
@@ -356,21 +350,17 @@ def __unbias_ngrams(self, ngram_length):
...
-                        if ngram_counts < self.__tfidf_matrix.data[start_idx_ptr + idx]:
-                            self.__tfidf_matrix.data[start_idx_ptr + idx] -= ngram_counts
-                        else:
-                            self.__tfidf_matrix.data[start_idx_ptr + idx] = 0
+                    self.__unbias_ngrams_slice(indices_slice, idx_ngram_minus_front, ngram_counts, start_idx_ptr)
+                    self.__unbias_ngrams_slice(indices_slice, idx_ngram_minus_back, ngram_counts, start_idx_ptr)
 
-                    if idx_ngram_minus_back in slice:
-                        idx = slice.tolist().index(idx_ngram_minus_back)
+    def __unbias_ngrams_slice(self, dindices_slice, idx_ngram_minus_front, ngram_counts, start_idx_ptr):
```
Comment (IanGrimstead) [inline scripts/algorithms/tfidf.py]: Change name from `idx_ngram_minus_front` as that's the same as one use - hmmn - `ngram_idx`?

Comment (thanasions) [reply scripts/algorithms/tfidf.py]: good point. That happens when pycharm does it for you :)

**Comment by thanasions** [comment]
Latest changes up on github now

**Pull request merged**